### PR TITLE
Issue links (get, delete)

### PR DESCRIPTION
**Pull request by yepninja**
List issue relations: https://gitlab.com/gitlab-org/gitlab/-/blob/master/doc/api/issue_links.md#list-issue-relations
Delete an issue link: https://gitlab.com/gitlab-org/gitlab/-/blob/master/doc/api/issue_links.md#delete-an-issue-link

### CODE REVIEW

File: packages/gitbeaker-core/src/services/Issues.ts [hunk 1]
```diff
@@ -69,6 +75,19 @@ export class Issues extends BaseService {
...
+  
+  deleteLink(
+    projectId: string | number,
+    issueIId: number,
+    issueLinkId: string | number,
+    options?: BaseRequestOptions,
+  ) {
+    const [pId, iId, iLinkId] = [projectId, issueIId, issueLinkId].map(encodeURIComponent);
+
+    return RequestHelper.delete(this, `projects/${pId}/issues/${iId}/links/${iLinkId}`, {
```
Comment (jdalrymple) [inline packages/gitbeaker-core/src/services/Issues.ts:87]: Use del instead of delete here

Comment (yepninja) [reply packages/gitbeaker-core/src/services/Issues.ts]: Fixed

File: packages/gitbeaker-core/src/services/Issues.ts [hunk 2]
```diff
@@ -69,6 +75,19 @@ export class Issues extends BaseService {
       ...options,
     });
   }
+  
+  deleteLink(
```
Comment (jdalrymple) [inline packages/gitbeaker-core/src/services/Issues.ts:79]: Just for consistency, Use `remove` instead of `delete`

Comment (yepninja) [reply packages/gitbeaker-core/src/services/Issues.ts]: Fixed

**Comment by jdalrymple** [comment]
LGTM, ill merge it up this week! Thank you :pray:

**Comment by fbridault** [comment]
Great I was looking for that !

**Pull request merged**